### DISCUSSION TITLE
XRDP

### DISCUSSION THREAD

**Discussion by rocheston** [trunkmaster/nextspace#384] [Q&A]
I installed XRDP and TigerVNCServer. Unable to see the desktop in XRDP sessions. Does Nextspace work with XRDP?

If yes, what settings should I configure for ~/.Xclients

**Comment by trunkmaster** [comment]
I don't know. I didn't test it.

**Comment by rocheston** [comment]
Can you tell me what desktop Nextspace runs? I can configure this in /etc/xrdp/startwm.sh and ~/.xsession

**Comment by trunkmaster** [comment]
What do you mean by "desktop"? If you mean window manager - it's /usr/NextSpace/Apps/Workspace.app/Workspace. But first it should start user level GNUstep services. I've written a script `gnustep-services`. So you can write into your `/etc/xrdp/startwm.sh` kind of this:
```
#!/bin/sh
/usr/NextSpace/bin/gnustep-services start
/usr/NextSpace/Apps/Workspace.app/Workspace
/usr/NextSpace/bin/gnustep-services stop
```

**Comment by rocheston** [comment]
Unable to get the XRDP to work. I want to upload the NeXTSPACE OS to the cloud and access using XRDP. If you could get it to work, please let me know. Thanks.

**Comment by rocheston** [comment]
XRDP: How do you configure .xsession? For example for XFCE desktop the .xsession entry is xfce4-session

**Comment by trunkmaster** [comment]
I've already answered to this question earlier:
```
#!/bin/sh
/usr/NextSpace/bin/gnustep-services start
/usr/NextSpace/Apps/Workspace.app/Workspace
/usr/NextSpace/bin/gnustep-services stop
```

**Comment by rcarmo** [comment]
This works great for me. Just rebuilt all the packages on Fedora 35 and added the lines above to my startwm script.
